setup.py

```diff
@@ -37,6 +37,7 @@
 INSTALL_REQUIRES = [
     "numpy",
     "pyserial>=3.3",
+    'pyvisa>=1.9.0,<1.10.0'
```

Comment:
Is there something in this code that doesn't work with `pyvisa>=1.10.0` ?

Reply:
I wanted to just include pyvisa 1.9.*, because I don't know if they are going to break some functionality in the next version (the following release could be either 1.10.0 or 2.0.0).

Reply:
Normally minor releases should not break compatibility. There is a selection syntax to stay on the same major release, but I don't remember off the top of my head. I got a feeling it's probably premature to place an upper limit, I'd do that only if difficulties arise.

Reply:
Agreed. I do not want to block someone from using >1.9 unless there is a known issue with a release.

Reply:
Ok, shall I put just `pyvisa>=1.9.0` then?

Reply:
Yeah